Dashboard/app/js/lib/controllers/permissions.js

```diff
@@ -176,6 +171,15 @@ FLOW.permControl = Ember.Controller.create({
       }
     }
 
+    // return superAdmin permissions
```

Comment:
I am wondering if we even need this whole section.  The idea behind it was to shortcut the whole checking of permissions for all the ancestors for superAdmins but it actually doesn't do and the permissions on the root folder i.e. with the `id === 0` will always happen

Reply:
True, removing the redundant code